test/simulator/timestamp/src/session_simulator.cpp

```diff
@@ -337,6 +363,7 @@ session_simulator::reset_txn_level_var()
     _ts_round_read = false;
     _prepared_txn = false;
     _txn_running = false;
+    _durable_ts_set = false;
 }
```

Comment:
Please A-Z this list of variables

Reply:
Updated in [4c0ea39](https://github.com/wiredtiger/wiredtiger/pull/8398/commits/4c0ea397095154f094db800a7e37d81fdb814983)